common/test-framework/org.jboss.tools.common.reddeer/src/org/jboss/tools/common/reddeer/propertieseditor/PropertiesEditor.java

```diff
@@ -0,0 +1,142 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
```

Comment:
All copyrights should be updated to include x-2017 range, eg., 2015-2017 as appropriate.